tarpc/Cargo.toml

```diff
@@ -28,6 +28,7 @@ travis-ci = { repository = "google/tarpc" }
 [dependencies]
 fnv = "1.0"
 futures = "0.3"
+futures-timer = "3.0"
```

Comment:
We've seen some issues come from this crate in the past just FYI. I would advise against using it.